tensorflow/python/training/experimental/loss_scale_optimizer.py

```diff
@@ -243,3 +243,8 @@ def _resource_apply_sparse(self, grad, handle, indices):
   def _resource_apply_dense(self, grad, handle):
     """This function should never be called."""
     raise RuntimeError('This function should never be called')
+
+  def variables(self):
+    """Returns the variables of the Optimizer."""
+    return self._optimizer.variables() + \
```

Comment:
Instead of using a backslash here, wrap the entire expression in parenthesis:

```
return (self._optimizer.variables() +
        list(self._loss_scale._weights.values())) # pylint: disable=protected-access
```

[Bashslashes aren't allowed by the style guide](http://google.github.io/styleguide/pyguide.html#32-line-length).

Reply:
changed